### PR TITLE
Batch update to improve readme descriptions

### PR DESCRIPTION
**Pull request by profexorgeek**
This PR fixes a lot of issues with yaml header descriptions using an automated tool. This replaces a prev branch that got out of sync with other bulk changes in master.

### CODE REVIEW

**Comment by davidbritch** [comment]
Wondering if an edit pass of each one is required, to eliminate sentences like:

For more information about this sample, see Dependency Resolution in Xamarin.Forms.
For more information about this sample, see Behaviors.
This sample is described in more detail in the article on BoxView.
To see the program built from the ground up, watch the video SkiaSharp Graphics for Xamarin Forms.

etc. etc.

**Comment by profexorgeek** [comment]
I could match on "For more information..." and strip sentences that start with that.

Other places where links are stripped but leave the linked text aren't necessarily bad. I think @conceptdev commented that to do many more edits we kinda need to see how these are going to be rendered. It could be that the description is truncated to a shorter length anyway!

**Comment by profexorgeek** [comment]
I tested stripping all sentences starting with "For more info" to the end of the doc. I think it's a net positive. In most cases it just strips the ending "For more information" sentence. But in a few cases it strips out a lot of stuff that shouldn't be in the description anyway. I couldn't find a way to link directly to it but if you look at the last-commit changes to the ADB2CAuth readme you'll see what I mean.

File: EmployeeDirectory/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 name: Employee Directory
-description: This contacts application is Xamarin.Forms port of the [existing pre-build app](http://xamarin.com/prebuilt/employeedirectory).  With Xamarin.Forms...
+description: 'This contacts application is Xamarin.Forms port of the existing pre-build app. With Xamarin.Forms you can use C or XAML to create layout and this sample app shows how to use both of them. To switch between user interface implementations open ~/EmployeeDirectory/EmployeeDirectoryUI/App.cs file and change uiImplementation variable. Switch to XAML: public static class App { //Change the following line to switch between XAML and C versions private static UIImplementation uiImplementation UIImplementation.Xaml; ... } Switch to C: public static class App { //Change the following line to switch between XAML and C versions private static UIImplementation uiImplementation UIImplementation.CSharp; ... } Nugets used in this solution: [PCLStorage][pclHref], [Xamarin.Forms][xFormsHref], [Microsoft.Bcl][mBclHref], [Microsoft.Bcl.Build][mBclBuildHref], [Microsoft.Net.Http][mNetHttpHref] Components used in this solution: [Xamarin.Social][xSocialHref]'
```
Comment (conceptdev) [inline EmployeeDirectory/README.md]: For the next iteration, maybe we can limit to the first paragraph so it doesn't pull in too much?

File: RpnCalculator/README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
... an alternative approach to coding an RPN calculator for Xamarin.Forms. RPN is based on a stack. Numbers are pushed on the stack by pressing the ENTER key. Unary operations (such as log and sin) pop a number from the stack, apply the operation, and push the result back on the stack. Binary operations (such as + and /) pop two numbers from the stack, perform the operation, and push the result on the stack. To perform the calculation 5 &x00D7; (3 + 4) &x2013; 2 press the following keys: 5 ENTER 3 ENTER 4 ENTER + &x00D7; 2 ENTER &x2013; The + operation adds 3 and 4, the times operation multiplies 5 and that result, and the minus operation subtracts 2 from that result. The layout of the keys appears twice in the MainPage.xaml file, separately for portrait mode and landscape mode. The MainPage.xaml.cs code-behind file switches between these two layouts based on the relative width and height of the page. The calculator logic is encapsulated in the RpnCalculatorViewModel.cs file. The XAML file and the ViewModel are linked through XAML-based data bindings, which are described in detail in the series of series of articles on Data Binding, and particularly the The Command Interface article.'
```
Comment (conceptdev) [inline RpnCalculator/README.md]: Maybe just the first para again here?

**Comment by conceptdev** [comment]
Oh shoot - merge conflicts 😬 @davidbritch @profexorgeek

**Comment by davidbritch** [comment]
This'll be because I deleted the AMA samples.

**Comment by profexorgeek** [comment]
@conceptdev merge conflicts are resolved

**Pull request merged**